messaging/kafka-streams-reactive-messaging/src/test/java/io/quarkus/ts/messaging/kafka/KafkaGratefulShutdownIT.java

```diff
@@ -37,6 +39,12 @@
     static RestService app = new RestService()
             .withProperty("kafka.bootstrap.servers", kafka::getBootstrapUrl)
             .withProperty("quarkus.kafka-streams.bootstrap-servers", kafka::getBootstrapUrl)
+            .withProperty("mp.messaging.outgoing.login-http-response-values.key.serializer", STRING_SERIALIZER)
```

Comment:
Why is this definition needed here?

Reply:
This is a reported issue: that I should mention in the description

https://github.com/quarkus-qe/quarkus-test-framework/issues/278

Basically, this test is using an app created from two classes:
```
 @QuarkusApplication(classes = { SlowTopicConsumer.class, SlowTopicResource.class })
    static RestService app = new RestService()
```
And is throwing an exception that should be investigated. 

One solution could be instantiate a regular RestService:
```
 @QuarkusApplication
    static RestService app = new RestService()
            .withProperty("kafka.bootstrap.servers", kafka::getBootstrapUrl)
            .withProperty("quarkus.kafka-streams.bootstrap-servers", kafka::getBootstrapUrl);
```

As we did in `StrimziKafkaStreamIT` or other scenarios, but by this way, we also tested the string serializer.

Reply:
This is not an issue of the test framework, but a possible issue in Quarkus, see my comment: https://github.com/quarkus-qe/quarkus-test-framework/issues/278#issuecomment-927562545

Reply:
Even if is not an issue we can't extend our current `application.properties` with custom properties because this`application.properties` is not compatible with the following scenario and auto-serializers: 

```
 @QuarkusApplication(classes = { SlowTopicConsumer.class, SlowTopicResource.class })
    static RestService app = new RestService()
            .withProperty("kafka.bootstrap.servers", kafka::getBootstrapUrl)
            .withProperty("quarkus.kafka-streams.bootstrap-servers", kafka::getBootstrapUrl)
```